molecule/data/solvation.py

```diff
@@ -390,44 +406,33 @@ def get_solvent_viscosity(self, T):
         """
         return math.exp(self.A + (self.B / T) + (self.C * math.log(T)) + (self.D * (T ** self.E)))
 
+    def get_solvent_density(self, T):
+        """
+        Returns the density of the solvent in Pa if the solvent is available in CoolProp (i.e. name_in_coolprop is
+        not None); raises DatabaseError if the solvent is not available in CoolProp (i.e. name_in_coolprop is None).
+        """
+        if self.name_in_coolprop is not None:
+            return PropsSI('D', 'T', T, 'Q', 0, self.name_in_coolprop)
+        else:
+            raise DatabaseError("Saturation pressure is not available for the solvent whose `name_in_coolprop` is None")
 
-    # def get_solvent_saturation_pressure(self, T):
-    #     """
-    #     Returns the saturation pressure of the solvent in Pa if the solvent is available in CoolProp (i.e. name_in_coolprop is
-    #     not None); raises DatabaseError if the solvent is not available in CoolProp (i.e. name_in_coolprop is None).
-    #     """
-    #     if self.name_in_coolprop is not None:
-    #         return PropsSI('P', 'T', T, 'Q', 0, self.name_in_coolprop)
-    #     else:
-    #         raise DatabaseError("Saturation pressure is not available for the solvent whose `name_in_coolprop` is None")
-    #
-    # def get_solvent_density(self, T):
-    #     """
-    #     Returns the density of the solvent in Pa if the solvent is available in CoolProp (i.e. name_in_coolprop is
-    #     not None); raises DatabaseError if the solvent is not available in CoolProp (i.e. name_in_coolprop is None).
-    #     """
-    #     if self.name_in_coolprop is not None:
-    #         return PropsSI('D', 'T', T, 'Q', 0, self.name_in_coolprop)
-    #     else:
-    #         raise DatabaseError("Saturation pressure is not available for the solvent whose `name_in_coolprop` is None")
-    #
-    # def get_solvent_coolprop_Tmin(self):
-    #     """
-    #     Returns the minimum temperature range available for the solvent in  CoolProp
-    #     """
-    #     if self.name_in_coolprop is not None:
-    #         return PropsSI('Tmin', self.name_in_coolprop)
-    #     else:
-    #         raise DatabaseError("Tmin is not available for the solvent whose `name_in_coolprop` is None")
-    #
-    # def get_solvent_coolprop_Tcrit(self):
-    #     """
-    #     Returns the critical temperature of solvent from CoolProp
-    #     """
-    #     if self.name_in_coolprop is not None:
```

Comment:
It looks like these are getting uncommented. I think these are commented to avoid requiring coolprop.

Reply:
A number of cases in this file.